src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -352,7 +352,7 @@ void MakeSureResultExists(Compound compound)
 
             result.Efficiency = efficiency;
 
-            speedFactor *= process.Rate;
+            speedFactor *= process.Rate * process.Count;
```

Comment:
I don't understand why this needs to be made this confusing. Why not just have an "EffectiveRate" property in the tweaked process class? That then can take into account the Rate and a maybe a variable called AdjustedRate or ControlledRate which is set by the process control (or even the player through the GUI in the future).

Reply:
Previously the rate was basically a count anyway. I basically did precisely what you just suggested with count as "rate" and rate (now set by the process control) as "EffectiveRate"